swift-sdk/Internal/LegacyRequestHandler.swift

```diff
@@ -225,30 +225,6 @@ class LegacyRequestHandler: RequestHandlerProtocol {
                               forResult: result)
     }
     
-    // MARK: DEPRECATED
-    
-    @discardableResult
-    func trackInAppOpen(_ messageId: String,
-                        onSuccess: OnSuccessHandler? = nil,
-                        onFailure: OnFailureHandler? = nil) -> Future<SendRequestValue, SendRequestError> {
-        let result = apiClient.track(inAppOpen: messageId)
-        return applyCallbacks(successHandler: onSuccess,
-                              andFailureHandler: onFailure,
-                              withIdentifier: "trackInAppOpen",
-                              forResult: result)
-    }
-    
-    @discardableResult
```

Comment:
This one did not had //deprecation Mark... is this change intentional?

Reply:
We only marked deprecation on the very top levels, but these calls were exclusive to their path/stack.